#shayan riyaz (#193)
bitprj/devrel: Hadoop/hadoop-proposal.md

> # TEMPLATE

Reviewer: remove the "template" and "do not edit" lines here
```suggestion
```

> ## :fire: Do not edit this file - copy the template and create your own file.

Reviewer: remove the "template" and "do not edit" lines here
```suggestion
```

**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**

### :pushpin: Step 1
**TITLE:**    
Understanding Hadoop

**TOPIC:**    
Apache Hadoop

**DESCRIPTION (5-7+ sentences):**    
The Apache Hadoop software library is a framework that allows for the distributed processing of large data sets across clusters of computers using simple programming models. It is designed to scale up from single servers to thousands of machines, each offering local computation and storage. Rather than rely on hardware to deliver high-availability, the library itself is designed to detect and handle failures at the application layer, so delivering a highly-available service on top of a cluster of computers, each of which may be prone to failures.

### :pushpin: Step 2
:family: **TARGET AUDIENCE (3-5+ sentences):**    

> We will be reaching out to people who want to get started in the world of big data but donot understand data processing distribution.

Reviewer: ```suggestion
We will be reaching out to people who want to get started in the world of big data but do not understand data processing distribution.
```
